pombola/writeinpublic/client.py

```diff
@@ -89,7 +89,10 @@ def get_messages(self, person_popolo_uri):
             'api_key': self.api_key,
             'person__popolo_uri': person_popolo_uri,
         }
-        response = requests.get(url, params=params)
-        response.raise_for_status()
-        messages = response.json()['objects']
-        return [Message(m, adapter=self.adapter) for m in messages]
+        try:
+            response = requests.get(url, params=params)
+            response.raise_for_status()
+            messages = response.json()['objects']
+            return [Message(m, adapter=self.adapter) for m in messages]
+        except requests.exceptions.RequestException as err:
```

Comment:
It worries me a bit that this is catching any error that might arise from the request, including things we might want to know about like 5xx errors. I think I'd be tempted to do something like:
```
try:
    response = requests.get(url, params=params)
    if response.status == 404:
        return []
    response.raise_for_status()
    messages = response.json()['objects']
    return [Message(m, adapter=self.adapter) for m in messages]
```

Then the calling code in `pombola/writeinpublic/views.py` wouldn't need to be changed as part of this PR...

Reply:
Ah yes you're quite right, this is being a bit over-eager in the errors it catches! Will change inline with your suggestion :+1: